v03_pipeline/lib/tasks/write_family_table.py

```diff
@@ -79,26 +79,20 @@ def create_table(self) -> hl.Table:
             ),
             False,
         )
-        ht = callset_mt.select_rows(
-            filters=callset_mt.filters.difference(self.dataset_type.excluded_filters),
-            entries=hl.sorted(
-                hl.agg.collect(
-                    hl.struct(
-                        s=callset_mt.s,
-                        **get_fields(
-                            callset_mt,
-                            self.dataset_type.genotype_entry_annotation_fns,
-                            **self.param_kwargs,
-                        ),
-                    ),
-                ),
-                key=lambda e: e.s,
+        ht = compute_callset_family_entries_ht(
+            self.dataset_type,
+            callset_mt,
+            get_fields(
+                callset_mt,
+                self.dataset_type.genotype_entry_annotation_fns,
+                **self.param_kwargs,
             ),
-        ).rows()
-        ht = ht.filter(ht.entries.any(self.dataset_type.sample_entries_filter_fn))
-        ht = globalize_sample_ids(ht)
+        )
```

Comment:
This small chunk of changes allows the family table structure to stay exactly the same.